specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -2222,6 +2270,10 @@
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
           "type": "object"
         },
+        "credString": {
```

Comment:
Here too, please use full words in names

Reply:
Same here. This field will be back-filled into connection string, so the name should be the same.